sebix789/AICaptchaPass
# feat: create OCR model with Attention mechanism

## Context
Create OCR model with Attention Mechanism.
[Jira](https://styralski98.atlassian.net/browse/SCRUM-13)

## Test Steps
- Pull this PR
- Run `python modelOCR.py`
- See if model is compiled and summary is generated

## Notes
Attention mechanism helps model with better text recognition
